README.md

```diff
@@ -68,6 +69,11 @@ format>
 
 Where:
 
+--jitter-uniform <duration>
+Add uniformly distributed absolute uniformly distributed
```

Comment:
(nit) Doesn't seem to read well (uniformly distributed repeats twice). Unless I am misunderstanding the sentence.

Reply:
Ah, I messed this up with a bad replacement, let me amend that.

Reply:
Done!